JuiceMaker/JuiceMaker/Controller/ManageViewController.swift

```diff
@@ -7,9 +7,61 @@
 
 import UIKit
 
-class ManageViewController: UIViewController {
+protocol ManageViewControllerDelegate {
```

Comment:
Delegate 네이밍 좋습니다! 그리고 해당 프로토콜을 클래스 객체 타입에서만 채택할 수 있도록 하려면 아래와 같이 작성해줄수 있습니다. 이렇게 하면 Delegate 패턴 활용을 위한 안정적인 프로토콜이 될 수 있어요!
```suggestion
protocol ManageViewControllerDelegate: AnyObject {
```